tasks.py

```diff
@@ -207,10 +207,11 @@ def mv_usage_stats_json(ctx):
         "name": "Docker image name.",
         "tag": "Docker image tag.",
         "build": "If True build the image, otherwise run it. Defaults to False.",
+        "detach": "Run container in background and print container ID. Defaults to False.",
```

Comment:
Do these changes and the `pyproject.toml` ones make sense in this PR? Can we separate them out?

Reply:
No they really don't. Was trying to just get them in here to prevent blocking the develop merge queue with minor PRs.

Reply:
I'd lean towards a separate PR if you don't mind 😄 I agree with the notion but CI should get through these invoke changes pretty quickly!